A3A/addons/ultimate/config/trader/cfgTraderMods.hpp

```diff
@@ -75,6 +75,11 @@ class traderMods
         addons[] = {"CUP_Creatures_People_Civil_Russia", "CUP_BaseConfigs", "CUP_AirVehicles_Core"};
         prefix = "cup";
     };
+    class csla : base
+    {
+        addons[] = {"CSLA"};
```

Comment:
is the cfgPatches for CSLA literally just "CSLA"?

Reply:
addons,yes, also US stuff uses US85 or something like that